src/sections/about-movement/index.module.css

```diff
@@ -0,0 +1,81 @@
+@import "../../styles/all.module.css";
+
+.root {
+  padding: 102px 24px;
+  background: $light-gray;
+
+  @include breakpoint-mobile-only {
+    padding: 72px 36px;
+  }
+
+  @include breakpoint-xs-only {
+    padding: 56px 36px;
+  }
+}
+
+.container {
+  max-width: 902px;
+  margin: 0 auto;
+}
+
+.author {
+  height: 48px;
+  text-align: center;
+
+  img {
+    height: 48px;
+  }
+}
+
+.title {
+  margin: 48px 0;
+  font-size: $font-size-h1;
```

Comment:
Why not use `variant="h1"` on `Typography`?

Reply:
Because the title isn't an `h1` 😉.

Reply:
But you have: `font-size: $font-size-h1;` 🙄